qpc/cred/add.py

```diff
@@ -68,10 +68,10 @@ def __init__(self, subparsers):
             help=_(messages.CRED_PWD_HELP),
         )
         group.add_argument(
-            "--sshkey",
-            dest="ssh_key",
-            action="store_true",
-            help=_(messages.CRED_SSH_KEY_HELP),
+            "--sshkeyfile",
+            dest="ssh_keyfile",
+            metavar="SSH_KEYFILE",
```

Comment:
What is `metavar` doing here?

Reply:
just telling the library to use SSH_KEYFILE for display/usage purposes instead of the default dest.